test/dotnet.Tests/GivenThatTheUserIsRunningDotNetForTheFirstTime.cs

```diff
@@ -117,10 +134,11 @@ public void ItDoesNotCreateAFirstUseSentinelFileUnderTheDotDotNetFolderWhenInter
             var homeFolder = new DirectoryInfo(Path.Combine(emptyHome, ".dotnet"));
             string[] fileEntries = Directory.GetFiles(homeFolder.ToString());
             fileEntries.Should().OnlyContain(x => !x.Contains(".dotnetFirstUseSentinel"));
+            fileEntries.Should().OnlyContain(x => !x.Contains(".aspNetCertificateSentinel"));
         }
 
         [Fact]
-        public void ItShowsTheTelemetryNoticeWhenInvokingACommandAfterInternalReportInstallSuccessHasBeenInvoked()
+        public void ItShowsTheTelemetryNoticeAndTheAspNetCertificateGenerationMessageWhenInvokingACommandAfterInternalReportInstallSuccessHasBeenInvoked()
```

Comment:
I would break this into two tests. One to assert on the telemetry notice. The other to assert on the Cert generation.